streams/src/main/java/org/apache/kafka/streams/kstream/internals/RepartitionedInternal.java

```diff
@@ -21,33 +21,33 @@
 import org.apache.kafka.streams.processor.StreamPartitioner;
 import org.apache.kafka.streams.processor.internals.InternalTopicProperties;
 
-class RepartitionedInternal<K, V> extends Repartitioned<K, V> {
+public class RepartitionedInternal<K, V> extends Repartitioned<K, V> {
```

Comment:
Must be public to be visible in Scala

Reply:
It's worth noting that it only needs to be visible for the scala _tests_ that verify the scala Repartitioned builder results in a correctly configured object. For the public API, we only convert a scala Repartitioned to a java Repartitioned.